0.2/docker-entrypoint.sh

```diff
@@ -60,15 +60,27 @@ if [ "$1" == 'janusgraph' ]; then
     fi
   done < <(env)
 
-  # wait for storage
-  if ! [ -z "${JANUS_STORAGE_TIMEOUT:-}" ]; then
-    F=$(mktemp --suffix .groovy)
-    echo "graph = JanusGraphFactory.open('${JANUS_CONFIG_DIR}/janusgraph.properties')" > $F
-    timeout ${JANUS_STORAGE_TIMEOUT}s bash -c \
-      "until bin/gremlin.sh -e $F > /dev/null 2>&1; do echo \"waiting for storage...\"; sleep 5; done"
-  fi
+  if [ "$2" == 'write-config' ]; then
+    echo "Config write into ${JANUS_CONFIG_DIR}"
+    exit 0
+  elif [ "$2" == 'print-config' ]; then
+    echo "Print $JANUS_PROPS"
```

Comment:
How about:

```
# contents of ${JANUS_PROPS}
```

with the literal `#` as that blends in with the config file following after it?